stdlib/grpc/src/main/java/org/ballerinalang/net/grpc/ClientCall.java

```diff
@@ -92,7 +92,7 @@ private void prepareHeaders(
                 x -> observerContext.ifPresent(ctx -> ctx.addTag(x.getKey(), x.getValue())));
 
         outboundMessage.setProperty(Constants.TO, "/" + method.getFullMethodName());
-        outboundMessage.setHttpMethod(GrpcConstants.HTTP_METHOD);
+        outboundMessage.setHttpMethod();
```

Comment:
Is this change the default behaviour? What is the difference with and without this?

Reply:
In gRPC, the HTTP method is always POST. Actually we don't need to ask for a method.  So I change the method and remove input arg. So when we call the method, it always set the POST method. 
This is no break existing code since this method only used internally.